test/csuite/time_shift_test.sh

```diff
@@ -42,10 +42,19 @@ then
     exit $EXIT_FAILURE
 fi
 
+# Find test_rwlock file from the wiredtiger home directory.
+WT_HOME_DIR=$(git rev-parse --show-toplevel)
+RW_LOCK_FILE=`find $WT_HOME_DIR -type f -name "test_rwlock" -print -quit`
```

Comment:
Why not just $WT_HOME_DIR/test/csuite/test_rwlock?

Reply:
Yes, it can be hardcoded to call the binary (test_rwlock) as it was in the script. But I have added a validation check to exit if the file doesn't exist.

Reply:
Then I think the validation check can also be removed.

Reply:
Removed